mailthon/api.py

```diff
@@ -28,8 +28,10 @@ def email(sender=None, receivers=(), cc=(), bcc=(),
     :param attachments: List of filenames to
         attach to the email.
     """
-    html = [HTML(content, encoding)]
-    files = [Attachment(k) for k in attachments]
+    enclosure=[]
+    enclosure.append(HTML(content, encoding))
+    enclosure.extend(tuple(Attachment(k) for k in attachments))
```

Comment:
This can be made more efficient by:

```python
enclosure = [HTML(content, encoding)]
enclosure.extend(Attachment(k) for k in attachments)
# etc.
```

Since it avoids one ``append`` method and also generates the Attachments lazily. A bit of an efficiency freak but :sweat_smile:.